internal/mithril-persistence/src/database/query/cardano_transaction/get_cardano_transaction.rs

```diff
@@ -180,4 +180,92 @@ mod tests {
             records
         );
     }
+
+    #[test]
+    fn above_slot_number() {
+        let connection = cardano_tx_db_connection().unwrap();
+
+        let cursor = connection
+            .fetch(GetCardanoTransactionQuery::above_slot_number(SlotNumber(
+                51,
+            )))
+            .unwrap();
+        assert_eq!(0, cursor.count());
```

Comment:
Maybe we can separate this test to make it more visible

Reply:
I have condensed the test to make if easier to read :+1: